09-gis.Rmd

```diff
@@ -618,87 +618,82 @@ In addition, if you would like to run simulations with the help of a geodatabase
-In this section we will only have a short glance at two selected examples.
-Hopefully, this will give an impression of all the spatial power and flexibility that is readily available on our fingertips.
+The focus of this chapter is on R interfaces to Desktop GIS software.
+We emphasize these bridges because dedicated GIS software is well-known and a common 'way in' to understanding spatial data, and also because they provide access to many geoalgorithms.
+It is important to be aware of other R bridges, to geospatial programs which based on command-line and are typically accessed via other programs through application programming interfaces (APIs).
+These 'other bridges' include interfaces to spatial libraries (section \@ref(gdal) covers R interfaces to GDAL), spatial databases (section \@ref(postgis)) and web mapping services (see chapter \@ref(adv-map)).
+This section provides only a snippet of what is possible with other bridges and, because of R's flexibility, many other bridges are possible.
+The aim is not to be comprehensive, but to demonstrate other ways of accessing the 'flexibility and power' in the quote by @sherman_desktop_2008 at the beginning of the chapter.
 
-### Spatial library API - GDAL {#gdal}
+### Bridges to GDAL {#gdal}
 
-GDAL is especially known for its support for numerous spatial drivers (data formats).
-In fact, there is barely a geospatial software that is not using GDAL in the background for importing and exporting geographic data.
-But GDAL offers more than I/O functions, e.g., geoprocessing tools for vector and raster data (for an overview have a look at http://www.gdal.org/pages.html).
-One can easily access these tools via the command line. 
-In the code chunk below `linkGDAL()` searches the computer for a working GDAL installation and adds the path of the executable files to the PATH variable. 
-This allows us to easily run GDAL via system calls.
-For example, `ogrinfo` gives back meta information of vector data.
+As discussed in Chapter \@ref(read-write), GDAL is a low-level library that supports many spatial data formats.
+GDAL is so effective that most GIS programs use GDAL in the background for importing and exporting geographic data, rather than re-inventing the wheel and using bespoke read-write code.
+But GDAL offers more than data I/O.
+It has [geoprocessing tools](http://www.gdal.org/pages.html) for vector and raster data, functionality to create [tiles](https://www.gdal.org/gdal2tiles.html) for serving raster data online, and rapid [rasterization](https://www.gdal.org/gdal_rasterize.html) of vector data, all of which can be accessed via the system of R command line.
+
+The code chunk below demonstrates this functionality:
+`linkGDAL()` searches the computer for a working GDAL installation and adds the location of the executable files to the PATH variable, allowing GDAL to be called on different operating systems.
+In the example below `ogrinfo` provides metadata above a vector dataset:
```

Comment:
metadata on, right?

Reply:
Yes that's a typo. well spotted! Will fix.

Reply:
Fixed.